feedback/static/html/feedback.html

```diff
@@ -0,0 +1,17 @@
+<div class="feedback_block">
+  <form action="" onsubmit="return false;">
+    <fieldset class="feedback_likert_field">
+      <legend class="feedback_likert_header">{likert_prompt}</legend>
+      <div class="feedback_likert_scale">{scale}</div>
+    </fieldset>
+
+    <div class="feedback_header_div">
+      <label class="feedback_header" for="feedback_freeform_textarea">{freeform_prompt}</label>
+    </div>
+    <div class="feedback_freeform_input">
+      <textarea name="feedback_freeform_textarea" class="feedback_freeform_area" rows="6" cols="45" placeholder="{placeholder}">{self.user_freeform}</textarea>
```

Comment:
placeholders are not appropriate for providing this sort of useful feedback.  Would it be possible to provide this feedback in a container below, then reference that container with an aria-describedby attribute on the textarea?  Same unique ID value issue would be applicable here as well.